cli/state.rs

```diff
@@ -48,14 +48,13 @@ pub struct State {
   pub global_state: ThreadSafeGlobalState,
   pub permissions: Arc<Mutex<DenoPermissions>>,
   pub main_module: ModuleSpecifier,
-  // TODO(ry) rename to worker_channels_internal
-  pub worker_channels: WorkerChannels,
   /// When flags contains a `.import_map_path` option, the content of the
   /// import map file will be resolved and set.
   pub import_map: Option<ImportMap>,
   pub metrics: Metrics,
   pub global_timer: Mutex<GlobalTimer>,
-  pub workers: Mutex<HashMap<u32, WorkerChannels>>,
+  pub workers: Mutex<HashMap<u32, WorkerChannelsExternal>>,
+  pub worker_channels_internal: Mutex<Option<WorkerChannelsInternal>>,
```

Comment:
Don't like that's it's behind a mutex, but I truly hope this is temporary

Reply:
Yes it’s temp. I should have added a TODO